dm/master/scheduler/scheduler.go

```diff
@@ -172,22 +177,12 @@ func (s *Scheduler) Start(pCtx context.Context, etcdCli *clientv3.Client) error
 
 	ctx, cancel := context.WithCancel(pCtx)
 
-	// starting to observe status of DM-worker instances.
-	workerEvCh := make(chan ha.WorkerEvent, 10)
-	workerErrCh := make(chan error, 10)
-	s.wg.Add(2)
-	go func() {
-		defer func() {
-			s.wg.Done()
-			close(workerEvCh)
-			close(workerErrCh)
-		}()
-		ha.WatchWorkerEvent(ctx, etcdCli, rev+1, workerEvCh, workerErrCh)
-	}()
-	go func() {
+	s.wg.Add(1)
+	go func(rev1 int64) {
 		defer s.wg.Done()
-		s.handleWorkerEv(ctx, workerEvCh, workerErrCh)
-	}()
+		// starting to observe status of DM-worker instances.
+		s.observeWorkerEvent(ctx, etcdCli, rev1)
```

Comment:
I prefer add a TODO here. Now `s.observeWorkerEvent` will not return other error because we don't return error in `s.handleWorkerEvent`. This can be done after we classify the retryable errors and fatal errors.

Reply:
OK